examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,123 @@
+from bcc import BPF
```

Comment:
I think it would help to chmod +x the flie that is checked in, as well as add `#/usr/bin/env python` as the first line.